app/kumactl/cmd/install/install_control_plane_test.go

```diff
@@ -316,6 +316,14 @@ controlPlane:
 			includeCRDs: true,
 			goldenFile:  "install-control-plane.gateway-api-present-not-enabled.yaml",
 		}),
+		Entry("should work with --set", testCase{
+			extraArgs: []string{
+				"--set",
+				"controlPlane.podSecurityContext.runAsUser=65000,controlPlane.podSecurityContext.runAsNonRoot=false",
+			},
+			includeCRDs: true,
+			goldenFile:  "install-control-plane.with-pod-security.yaml",
+		}),
```

Comment:
If you want as it's a little easier you can have new tests by just adding values files in `app/kumactl/cmd/install/testdata/install-cp-helm` I find this way it's also easier to look at test cases

Reply:
Will do!